apps/mdn/mdn-aws/k8s/configure_elb.sh

```diff
@@ -58,6 +58,14 @@ add_logging() {
         --region ${AWS_REGION}
 }
 
+configure_elb_timeout() {
+    echo "Setting ELB idle timeout"
+    aws elb modify-load-balancer-attributes \
+        --load-balancer-name ${ELB_NAME} \
+        --load-balancer-attributes "{\"ConnectionSettings\":{\"IdleTimeout\":120}}" \
```

Comment:
@metadave It would be nice (but of course not necessary) to somehow automatically keep this value in sync with `WEB_GUNICORN_TIMEOUT`. I wonder if it would be worthwhile to re-run `configure_elb_timeout()` whenever we re-deploy, just in case `WEB_GUNICORN_TIMEOUT` changes?

Reply:
I've pushed an update that uses `WEB_GUNICORN_TIMEOUT` to set the ELB timeout, as well as a comment above the `WEB_GUNICORN_TIMEOUT` definition mentioning that the ELB needs to be updated if the value changes.